src/main/java/ui/IdGenerator.java

```diff
@@ -19,6 +19,14 @@ public static String getRepositorySelectorId() {
         return "repositorySelector";
     }
 
+    public static String getRepositoryPickerTextFieldId() {
+        return "repositoryPickerUserInputField";
+    }
+
+    public static String getRepositoryPickerSuggestedRepoListId() {
+        return "suggestedRepositoryList";
+    }
+
```

Comment:
I think so, but the original file groups `getObjectId` methods together (similarly for `getObjectReference`). Do you think we should move them as well?